keps/sig-windows/0000-20181221-windows-group-managed-service-accounts-for-container-identity.md

```diff
@@ -110,31 +110,76 @@ When this app moves to production on containers, the team will need to coordinat
   - Provision the GMSA and gives details to Kubernetes cluster admin.
   - Assign access to a AD security group to control what machines can use this GMSA. The AD security group should include all authorized Kubernetes nodes.
 
-2. A Kubernetes cluster admin (or cluster setup tools or an operator) will install a cluster scoped CRD (of kind GMSACredSpec) whose instances will be used to store GMSA credential spec configuration:
+2. A Kubernetes cluster admin (or cluster setup tools or an operator) will install a cluster scoped CRD (of kind GMSACredentialSpec) whose instances will be used to store GMSA credential spec configuration:
 
 ```
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: gmsacredspecs.auth.k8s.io
+  name: gmsacredentialspecs.windows.k8s.io
 spec:
-  group: auth.k8s.io
+  group: windows.k8s.io
   version: v1alpha1
   names:
-    kind: GMSACredSpec
-    plural: gmsacredspecs
+    kind: GMSACredentialSpec
+    plural: gmsacredentialspecs
   scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        credspec:
+          description: GMSA Credential Spec
+          properties:
+            ActiveDirectoryConfig:
+              description: Windows Active Directory configuration
+              properties:
+                GroupManagedServiceAccounts:
+                  description: GMSA names and scopes
+                  type: array
+                  items:
+                    Name:
+                      description: Name of the primary GMSA or additional accounts
+                      type: string
+                    Scope:
+                      description: DNSRoot or NetBIOS name of the above GMSA
+                      type: string
+                CmsPlugins:
+                  description: Always specify a single item ActiveDirectory
+                  type: array
+                  items:
+                    ActiveDirectory
+                DomainJoinConfig:
+                  description: Configuration to join AD domain
+                  properties:
+                    DnsName:
+                      description: Name of AD domain forest
+                      type: string
+                    DnsTreeName:
+                      description: DNS root of AD domain
+                      type: string
+                    Guid:
+                      description: GUID of AD domain
+                      type: string
+                    MachineAccountName:
+                      description: Name of GMSA
+                      type: string
+                    NetBiosName:
+                      description: NetBios name of AD Domain
```

Comment:
in your example below, you mentioned the netbiosname being "demo". since it is the netbios of the domain, the example should list "contoso" instead

Reply:
Good point .. will update.

Reply:
Done.